plugins/docsettingtweak.koplugin/directory_defaults.lua

```diff
@@ -0,0 +1,33 @@
+return {--do NOT change this line
+
+ --HELP:
+ -- you can override default settings for documents per directory in this file
+ -- the directories must be under the home folder.
+ -- you can find settings to change in the metadata.lua in the .sdr folder.
+
+ -- syntax:
+
+ --   ["path/to/folder"] = {
+ --         ["setting_to_override"] = value,
+ --    },
+
+ -- examples:
+
+ --[[
+    ["/mnt/us/documents/hebrew"] = {
```

Comment:
How would it do if I add a directory here with a trailing / (eg. `["/mnt/us/documents/hebrew/"]`) ?
(If it wouldn't work, either make it work :), or add some comment here that a trailing slash should not be included.)